packages/rx_bloc_cli/example/docs/continuous_delivery.md

```diff
@@ -23,7 +23,24 @@ export MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD=<PASSWORD>
 
 Before committing any changes, make sure you encrypt all the files using the `encode.sh` script.
 
-### Fastfile amendments
+### Project & Fastfile amendments
+
+For the iOS project, make sure you set project signing to manual and apply expected provisioning profile for each build configuration.
+To do that, you need to:
+- Open project in XCode
+- Inside the targets section, select your `Runner` target
+- Open the `Signing & Capabilities` tab
+- Uncheck the `Automatically manage signing` checkmark in each of the build configurations
+- Select a fitting provisioning profile for each of the build configurations
```

Comment:
This is not true. Only for the release  configurations the automatic signing needs to be turned off